test/models/test_flava.py

```diff
@@ -126,3 +131,124 @@ def test_forward_pretraining(self):
             10.8777,
             places=4,
         )
+
+
+class TestFLAVAModel(unittest.TestCase):
+    def setUp(self):
+        self.text_encoder = flava_text_encoder(
+            hidden_size=2,
+            num_attention_heads=1,
+            num_hidden_layers=1,
+            hidden_dropout_prob=0.0,
+            intermediate_size=2,
+        )
+        self.image_encoder = flava_image_encoder(
+            hidden_size=2,
+            num_attention_heads=1,
+            num_hidden_layers=1,
+            hidden_dropout_prob=0.0,
+            intermediate_size=2,
+            image_size=2,
+            patch_size=1,
+            num_channels=3,
+            use_image_masking=True,
+        )
+
+        mm_encoder = nn.Identity()
+        image_to_mm_projection = nn.Identity()
+        text_to_mm_projection = nn.Identity()
+        self.flava = FLAVAModel(
+            image_encoder=self.image_encoder,
+            text_encoder=self.text_encoder,
+            mm_encoder=mm_encoder,
+            image_to_mm_projection=image_to_mm_projection,
+            text_to_mm_projection=text_to_mm_projection,
+        )
+
+    def _assert_empty(self, field):
```

Comment:
would it be useful for future testing to make this general for any of our model outputs? something like so:
'''
self.assertTrue(all(map(lambda x: x is None, field)))
'''
and then we could move to `test_utils.py`

Reply:
lets do it if / when we need it :)